frappe/installer.py

```diff
@@ -115,78 +106,132 @@ def install_db(root_login="root", root_password=None, db_name=None, source_sql=N
 
 	frappe.flags.in_install_db = False
 
+def plugin_site_specifics_from_app(app: str, verbose=False) -> None:
```

Comment:
why rename this function? `plugin_site_specifics_from_app` seems very cryptic

Reply:
`install_app_for_tenant` ?

Reply:
common fixtures can be added via `install_app`
tentant specific fixtures can be added via `install_tenant`

by default, we will install one tenant

Reply:
you mean `install_app_for_site` here?
 
I thought of it but I was unsure about having `install_app_for_tenant` and `install_app_for_site` names as it din't feel correct to me. will fix that!